### ISSUE TITLE
versionCode does not increment

### ISSUE THREAD

**Issue by marcopadillo** [novom/react-native-set-version#12]
Using the commands in the example [here](https://github.com/novom/react-native-set-version#how-the-version-code-and-cfbundleversion-are-updated), the expected results for the versionCode when executing `yarn set-version 1.0.0-rc.1` and `yarn set-version 1.0.0-rc.2` are `100001` and `100002` respectively. Here is what happened when I tried it on one of my projects:

```
$ yarn set-version 1.0.0-rc.1
# ...
# Will set Android version to 1.0.0-rc.1
# Will set Android version code to 100001
# Version replaced in build.gradle
# ...
```
```
$ yarn set-version 1.0.0-rc.2
# ...
# Will set Android version to 1.0.0-rc.2
# Will set Android version code to 100001
# Version replaced in build.gradle
# ...
```

**Comment by ssideleau** [comment]
Thank you for reporting the issue. I'll take a look at it in the following week.

**Comment by ssideleau** [comment]
This issue has been resolved in version 1.1.1
The version is available on npm: https://www.npmjs.com/package/react-native-set-version
